src/libstd/sys/unix/fd.rs

```diff
@@ -117,10 +113,6 @@ impl FileDesc {
     }
 
     pub fn write_at(&self, buf: &[u8], offset: u64) -> io::Result<usize> {
-        #[cfg(target_os = "android")]
-        use super::android::cvt_pwrite64;
-
-        #[cfg(not(target_os = "android"))]
         unsafe fn cvt_pwrite64(
```

Comment:
If you'll forgive the drive-by-comment, a lot of the organization in these idioms isn't necessary any longer now that the `#[cfg]` switch is gone, so you can probably clean these all up by inlining functions instead of declaring inline functions or inline blocks. Not critical of course, but just a thought!

Reply:
That absolutely makes sense, drive-by comment happily taken.
First I need to get it working, then I can make it nice.